### ISSUE TITLE
JSON gets truncated for two rules for some policies

### ISSUE THREAD

**Issue by o-l-a-v** [tyconsulting/AzPolicyTest#48]
```text
WARNING: Resulting JSON is truncated as serialization has exceeded the set depth of 2.
   [+] Policy rule existenceCondition must not be empty 2ms (2ms|1ms)
```

and

```text
WARNING: Resulting JSON is truncated as serialization has exceeded the set depth of 2.
   [+] Policy rule existenceCondition must not be empty 1ms (1ms|0ms)
```

Either hide the warning or use greater `-Depth`?

Example policy definition:

```json
{
  "$schema": "https://raw.githubusercontent.com/Azure/enterprise-azure-policy-as-code/main/Schemas/policy-definition-schema.json",
  "name": "f1055cad-206f-4d96-9b97-a64abb0f2aee",
  "properties": {
    "displayName": "Deploy Qualys Vulnerability Assessment Solution on Virtual Machines",
    "description": "This policy deploys the Qualys vulnerability assessment extension to Windows and Linux virtual machines.",
    "mode": "All",
    "metadata": {
      "category": "Dataprotection",
      "version": "1.0.0",
      "RemediationDescription": "Remediate through Azure Policy portal",
      "Severity": "Low"
    },
    "version": "1.0.0",
    "parameters": {
      "effect": {
        "allowedValues": [
          "DeployIfNotExists",
          "AuditIfNotExists",
          "Disabled"
        ],
        "defaultValue": "DeployIfNotExists",
        "metadata": {
          "description": "Enable or disable the execution of the policy.",
          "displayName": "Effect"
        },
        "type": "string"
      }
    },
    "policyRule": {
      "if": {
        "field": "type",
        "in": [
          "Microsoft.Compute/virtualMachines",
          "Microsoft.ClassicCompute/virtualMachines"
        ]
      },
      "then": {
        "effect": "[parameters('effect')]",
        "details": {
          "evaluationDelay": "PT10M",
          "type": "Microsoft.Compute/virtualMachines/extensions",
          "existenceCondition": {
            "allOf": [
              {
                "field": "Microsoft.Compute/virtualMachines/extensions/type",
                "in": [
                  "WindowsAgent.AzureSecurityCenter",
                  "LinuxAgent.AzureSecurityCenter"
                ]
              },
              {
                "field": "Microsoft.Compute/virtualMachines/extensions/publisher",
                "equals": "Qualys"
              },
              {
                "field": "Microsoft.Compute/virtualMachines/extensions/provisioningState",
                "equals": "Succeeded"
              }
            ]
          },
          "roleDefinitionIds": [
            "/providers/microsoft.authorization/roleDefinitions/fb1c8493-542b-48eb-b624-b4c8fea62acd"
          ],
          "deployment": {
            "properties": {
              "parameters": {
                "vmName": {
                  "value": "[field('name')]"
                },
                "apiVersionByEnv": {
                  "value": "2015-06-01-preview"
                }
              },
              "mode": "Incremental",
              "template": {
                "outputs": {},
                "parameters": {
                  "vmName": {
                    "type": "string"
                  },
                  "apiVersionByEnv": {
                    "type": "string"
                  }
                },
                "contentVersion": "1.0.0.0",
                "resources": [
                  {
                    "type": "Microsoft.Compute/virtualMachines/providers/serverVulnerabilityAssessments",
                    "apiVersion": "[parameters('apiVersionByEnv')]",
                    "name": "[concat(parameters('vmName'), '/Microsoft.Security/default')]"
                  }
                ],
                "variables": {},
                "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#"
              }
            }
          }
        }
      }
    }
  }
}
```

**Comment by tyconsulting** [comment]
this looks like a quick fix. let me update it...

**Comment by tyconsulting** [comment]
@o-l-a-v should be fixed now with version 2.7.2
